stable/mission-control/README.md

```diff
@@ -330,6 +389,11 @@ The following table lists the configurable parameters of the mission-control cha
 | `postgresql.slave.nodeSelector`              | PostgreSQL slave node selector                  | `{}`                                  |
 | `postgresql.slave.affinity`                  | PostgreSQL slave node affinity                  | `{}`                                  |
 | `postgresql.slave.tolerations`               | PostgreSQL slave node tolerations               | `[]`                                  |
+| `common.customInitContainers`                | Custom init containers                          | see [values.yaml](values.yaml)        |
+| `common.customVolumes`                       | Custom Volumes                                  | see [values.yaml](values.yaml)        |
+| `common.customVolumeMounts`                  | Custom Volume Mounts                            | see [values.yaml](values.yaml)        |
+| `common.configMaps`                          | Custom configMaps                               | see [values.yaml](values.yaml)        |
+| `common.customSidecarContainers`             | Custom SidecarContainers                        | see [values.yaml](values.yaml)        |
 | `database.type`                              | External database type (`postgresql`)           | `postgresql`                          |
 | `database.driver`                            | External database driver                        | `org.postgresql.Driver`               |
```

Comment:
Please add the newly added values to the table - `router.tlsEnabled` and move `uid` and `gid` to common

Reply:
done